examples/src/omezarr/omezarr-viewer.tsx

```diff
@@ -79,15 +151,36 @@ export function OmezarrViewer({
                                 color: [0, 0, 0, 0],
                                 depth: 1,
                             });
+                            lowResPreview(e.target, server.cache, (_e) => {})?.cancelFrame(
```

Comment:
here, whenever we begin a frame, we first:
1. request a low-resolution preview frame be rendered to our target. By immediately cancelling, we ensure that we only render things that are already in the cache. If they're not in the cache, they'll still be requested, so maybe they'll be present in the future.
2. on line 157, we render the stashed image ABOVE the low-res preview. This hides the "pixelated flickering" that would otherwise be caused by the low-res preview.
3. finally, we proceed as normal, rendering the actual frame we want above both the low-res preview and the stashed copy.